prisma/schema.prisma

```diff
@@ -27,7 +27,7 @@ model User {
   skills       String[]      @default([])
   applications Application[]
   offers       Offer[]
-  company      Company?      @relation(fields: [companyId], references: [id])
+  company      Company?      @relation(fields: [companyId], references: [id], onDelete: Cascade)
```

Comment:
This will cause the user to be deleted along with the company. 
```suggestion
  company      Company?      @relation(fields: [companyId], references: [id])
```

Reply:
hehe, good catch, will commit your suggestion!